e2e-tests/insurance/cypress/e2e/journeys/policy/multiple-contract-policy/export-value/multiple-contract-policy-export-value-alternative-currency.spec.js

```diff
@@ -36,12 +36,11 @@ context('Insurance - Policy - Multiple contract policy - Export value page - Alt
     cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
       referenceNumber = refNumber;
 
-      cy.startInsurancePolicySection({});
-      cy.completeAndSubmitPolicyTypeForm({ policyType });
-
-      cy.completeAndSubmitMultipleContractPolicyForm({
-        isoCode: NON_STANDARD_CURRENCY_CODE,
-        alternativeCurrency: true,
+      cy.completeAndSubmitPolicyForms({
+        formToStopAt: 'multipleContractPolicy',
+        policyType,
+        policyIsoCode: NON_STANDARD_CURRENCY_CODE,
+        policyAlternativeCurrency: true,
```

Comment:
Can we keep the same param names? `isoCode`, `alternativeCurrency`. Will simplify some things in the new command